timetracker/timetracker/urls.py

```diff
@@ -17,17 +17,11 @@
 from django.contrib import admin
 from django.http import HttpResponse
 
-
-def hello_world(request):
-    name = request.GET.get('name')
-    if name:
-        return HttpResponse('Hello %s' % name)
-    else:
-        return HttpResponse('<form><input name="name"></form>')
-
-
+import entries.views
```

Comment:
I recommend to stay consistent with relative and absolute imports of your own code. In other words, in `timetracker/entries/views.py` you use `from .models import ...`. You can do that here too: `from .views import clients, home, projects`.